src/examples/demo.ts

```diff
@@ -22,6 +22,43 @@ document.addEventListener('DOMContentLoaded', () => {
                     return '#046380';
                 }
             },
+            customLinkStyle: link => {
+                return {
+                    attrs: {
+                        '.connection': {
+                            stroke: '#3c4260',
+                            'stroke-width': 2
+                        },
+                        '.marker-source': {
+                            fill: '#4b4a67',
+                            stroke: '#4b4a67',
+                            d: 'M0,3a3,3 0 1,0 6,0a3,3 0 1,0 -6,0'
+                        },
+                        '.marker-target': {
+                            fill: '#4b4a67',
+                            stroke: '#4b4a67',
+                            d: 'M5.5,15.499,15.8,21.447,15.8,15.846,25.5,21.447,25.5,9.552,15.8,15.152,15.8,9.552z'
+                        }
+                    },
+                    labels: [
+                        {
+                            position: 0.5,
```

Comment:
Value `z: 0` is not default.
If value `position: 0.5` is not set, errors appear in jointjs.